### PR TITLE
Change `owner` to `active` in contract actions for new period.

### PR DESCRIPTION
**Pull request by dallasjohnson**
Changes permission reference to owner to active on the dacauthority permissions.

### CODE REVIEW

**Review by michaeljyeates — COMMENTED** [review]
Why is this removing the config struct?

File: newperiod_components.cpp [hunk 1]
```diff
@@ -140,12 +140,12 @@ void daccustodian::setCustodianAuths() {
...
     };
 
-    action(permission_level{accountToChange, "owner"_n},
+    action(permission_level{accountToChange, "active"_n},
            "eosio"_n, "updateauth"_n,
            std::make_tuple(
                    accountToChange,
                    HIGH_PERMISSION,
-                   "owner"_n,
+                   "active"_n,
```
Comment (michaeljyeates) [inline newperiod_components.cpp:148]: This is correct

File: newperiod_components.cpp [hunk 2]
```diff
@@ -140,12 +140,12 @@ void daccustodian::setCustodianAuths() {
             .accounts = accounts
     };
 
-    action(permission_level{accountToChange, "owner"_n},
+    action(permission_level{accountToChange, "active"_n},
```
Comment (michaeljyeates) [inline newperiod_components.cpp]: This should be owner still

Comment (dallasjohnson) [reply newperiod_components.cpp]: Do you mean this for each of the inline actions or just this one?

Comment (dallasjohnson) [reply newperiod_components.cpp]: Changed it back to `owner`

File: newperiod_components.cpp (hunk 1 above)
Comment (dallasjohnson) [reply newperiod_components.cpp:148]: 👍

**Review by michaeljyeates — APPROVED** [review]

**Pull request merged**